hack/fixup-csv

```diff
@@ -146,14 +149,13 @@ def main():
             args = [
                 a for a in container["args"]
                 if not a.startswith((
-                        "--dynamic-serving-dns-names",
+                        "--dynamic-serving-",
                         "--tls-cert-file",
                         "--tls-private-key-file",
                 ))
             ]
             # Add the arguments back with the modified values
             args.extend([
-                "--dynamic-serving-dns-names=cert-manager-webhook-service.$(POD_NAMESPACE).svc",
```

Comment:
I don't know why I left this particular argument in previous releases, but in 1.14 it is no longer allowed to mix the dynamic and the static certificate arguments, so I've removed it.
* https://github.com/cert-manager/cert-manager/pull/6574